<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather &amp; Bitcoin Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="weatherTab">
       <attribute name="title">
        <string>Weather</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>City:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cityInput">
            <property name="placeholderText">
             <string>Enter city name...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getWeatherBtn">
            <property name="text">
             <string>Get Weather</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="weatherGroupBox">
          <property name="title">
           <string>Weather Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Temperature:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="tempLabel">
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="descLabel">
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Humidity:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="humidityLabel">
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Wind Speed:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="windLabel">
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bitcoinTab">
       <attribute name="title">
        <string>Bitcoin Prices</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="sourceCombo">
            <item>
             <property name="text">
              <string>All Sources</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nobitex</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kraken</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CoinMarketCap</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Binance</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Coinbase</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blockchain</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getBitcoinBtn">
            <property name="text">
             <string>Get Bitcoin Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="autoRefreshBtn">
            <property name="text">
             <string>Auto Refresh: OFF</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="priceTable">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Source</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price (USD)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Update</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Ready</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>